src/platform.js

```diff
@@ -3,6 +3,22 @@ class Platform {
         Platform.OS = platform.name();
         Platform.Version = platform.version();
     }
+
+    static setLinking(linking) {
+      Platform.linking = linking;
+    }
+
+    static canOpenUrl(url) {
```

Comment:
doesn't this need to be async?

Reply:
This function returns a promise so it does not need to be async. We can make it async, but we will need to change their body - sth like this:
```
static async canOpenUrl(URL) {
    try {
        const result = await Platform.linking.canOpenURL(url);
        return result;
    } catch {
        throw new Error('Invalid format.');
    }
}
```
Check [this page](https://javascript.info/async-await).